docs/reference/rest-api/cron-expressions.asciidoc

```diff
@@ -0,0 +1,182 @@
+[[cron-expressions]]
+=== Cron expressions
+
+{es} uses the cron parser from the http://www.quartz-scheduler.org[Quartz Job Scheduler]. For more
+information about writing Quartz cron expressions, see the
+http://www.quartz-scheduler.org/documentation/quartz-2.2.x/tutorials/tutorial-lesson-06.html[Quartz CronTrigger Tutorial].
+
+All schedule times are in UTC, other timezones are not supported.
+
+A cron expression is a string of the following form:
+
+[source,txt]
+------------------------------
+    <seconds> <minutes> <hours> <day_of_month> <month> <day_of_week> [year]
+------------------------------
```

Comment:
This seems like a better fit at the very top of the page. No big deal either way though.